Resources/Prototypes/Catalog/Bounties/bounties.yml

```diff
@@ -548,8 +548,221 @@
   description: bounty-description-food
   idPrefix: UNTH
   entries:
-  - name: bounty-food
+  - name: bounty-item-food
     amount: 30
     whitelist:
       tags:
       - Meat
+
+- type: cargoBounty
+  id: BountyFruit
+  reward: 5000
+  description: bounty-description-fruit
+  entries:
+  - name: bounty-item-fruit
+    amount: 12
+    whitelist:
+      tags:
+      - Fruit
+
+- type: cargoBounty
+  id: BountyVegetable
+  reward: 6000
+  description: bounty-description-vegetable
+  entries:
+  - name: bounty-item-vegetable
+    amount: 14
+    whitelist:
+      tags:
+      - Vegetable
+
+- type: cargoBounty
+  id: BountyManlyDorf
+  reward: 3300
+  description: bounty-description-manly-dorf
+  entries:
+  - name: bounty-item-manly-dorf
+    amount: 4
+    whitelist:
+      tags:
+      - ManlyDorf
+
+- type: cargoBounty
+  id: BountyChili
+  reward: 5555
+  description: bounty-description-chili
+  entries:
+  - name: bounty-item-chili
+    amount: 3
+    whitelist:
+      tags:
+      - ChiliBowl
+
+# If someone else wants to do this, go for it, making a bounty check for alive seems like a headache
+
+# - type: cargoBounty
+#  id: BountyLiveMouse
+#  reward: 1500
+#  description: bounty-description-live-mouse
+#  entries:
+#  - name: bounty-item-live-mouse
+#    amount: 1
+#    whitelist:
+#      tags:
+#     -
+
+- type: cargoBounty
+  id: BountyRollerskates
+  reward: 6500
+  description: bounty-description-rollerskates
+  entries:
+  - name: bounty-item-rollerskates
+    amount: 2
+    whitelist:
+      components:
+      - Skates
+
+- type: cargoBounty
+  id: BountyBedsheet
+  reward: 4100
+  description: bounty-description-bedsheet
+  entries:
+  - name: bounty-item-bedsheet
+    amount: 5
+    whitelist:
+      tags:
+      - Bedsheet
+
+- type: cargoBounty
+  id: BountyBandana
+  reward: 4000
+  description: bounty-description-bandana
+  entries:
+  - name: bounty-item-bandana
+    amount: 7
+    whitelist:
+      tags:
+      - Bandana
+
+- type: cargoBounty
+  id: BountyCookedMeat
+  reward: 3200
+  description: bounty-description-cooked-meat
+  entries:
+  - name: bounty-item-cooked-meat
+    amount: 4
+    whitelist:
+      tags:
+      - Cooked
+      - Meat
+
+- type: cargoBounty
+  id: BountyBanana
+  reward: 6009
+  description: bounty-description-banana
+  entries:
+  - name: bounty-item-banana
+    amount: 9
+    whitelist:
+      tags:
+      - Banana
+
+- type: cargoBounty
+  id: BountyBeer
+  reward: 3100
+  description: bounty-description-beer
+  entries:
+  - name: bounty-item-beer
+    amount: 6
+    whitelist:
+      tags:
+      - Beer
+
+- type: cargoBounty
+  id: BountyHi-VizVest
+  reward: 3030
+  description: bounty-description-hi-viz-vest
+  entries:
+  - name: bounty-item-hi-viz-vest
+    amount: 3
+    whitelist:
+      tags:
+      - Hi-Viz
```

Comment:
dont name tags like this, or the prototype itself

Reply:
How would you like them to be named?

Reply:
HiVizVest (hi viz hats when)

dont include - or _ in prototype names have it PascalCase